templates/jobs/email_templates/status.txt

```diff
@@ -2,5 +2,10 @@ Hello there!
 
 This is an automatically generated email about your {{ option }}. Current status is: {{ status }}.
 
+{% if reviewers_comment %}
+    This is your feedback:
+    {{ reviewers_comment|linebreaks }}
+{% endif %}
```

Comment:
Review with maildump if you can to see how this formats.  I think text emails will have extra line breaks where the template tags are, and will format with the indents.  Just make sure it looks how you want it to.